### PR TITLE
[#1917] Commits view: sort everything in reverse chronological order

### PR DESCRIPTION
**Pull request by pratham31012002**
This submission is made to fix https://github.com/reposense/RepoSense/issues/1917.

Proposed Commit Message:

```
Currently days are sorted in reverse chronological order while commits 
are sorted in chronological order.

Sorting days and commits in the same chronological order seems 
more intuitive.

Let's implement changes such that days and commits are sorted in 
the same chronological order. Sorting by LoC also sorts the commits 
of the day by LoC.
```

Current: Days are sorted in reverse chronological order while commits are sorted in chronological order.

After fix: Days and commits are sorted in the same chronological order. Sorting by LoC also sorts the commits of the day by LoC.

Illustration:
1) 2021-08-18 when sort by `Time`, order `Ascending`
image

2) 2021-08-18 when sort by `Time`, order `Descending`
image

3) 2021-08-18 when sort by `LoC`, order `Descending`
image

4) 2021-08-18 when sort by `LoC`, order `Ascending`
image

### CODE REVIEW

File: frontend/cypress/tests/chartView/chartView_mergeCommits.cy.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 describe('include merge commits in chart view', () => {
   it('show merge commits in summary chart', () => {
     // ramp chart should have merge commit slices
-    cy.get('[title="[2023-03-03] Merge branch \'new-branch\' into cypress: +0 -0 lines "]')
+    cy.get('[title="[2023-03-04] Merge branch \'new-branch\' into cypress: +0 -0 lines "]')
```
Comment (ckcherry23) [inline frontend/cypress/tests/chartView/chartView_mergeCommits.cy.js:4]: Thanks for taking a look into this @pratham31012002. The date `2023-03-03` may not pass Cypress tests in your local machine but is the date that is required for the CI frontend tests to pass. The discrepancy is probably due to a time zone issue as the commit was made close to midnight.

Comment (pratham31012002) [reply frontend/cypress/tests/chartView/chartView_mergeCommits.cy.js:4]: Oh okay thank you, I was wondering why I had to make that change, now it makes sense 👍

**Review by ckcherry23 — APPROVED** [review]
Thanks for your first PR @pratham31012002! Code changes LGTM. 
Just remember to update the commit message such that each line has not more than 72 characters.

**Comment by pratham31012002** [comment]
Kindly requesting your attention to review this pending PR at your earliest convenience.

**Review by sikai00 — APPROVED** [review]

**Comment by pratham31012002** [comment]
> Hi @pratham31012002, apologies as there appears to be a new frontend test `zoomView/zoomView_diffstat.cy.js` merged after the last CI run that fails. Could you take a look at it?

Hey I fixed this test case, Could you have a look?

**Comment by chan-j-d** [comment]
Thanks @pratham31012002 for fixing it. LGTM!

**Pull request merged**